src/pages/HomePage/Home.tsx

```diff
@@ -5,12 +5,12 @@ import SliderComponent from "../../components/ui/slider/Slider";
 import { BreedType } from "../../types";
 
 function Home() {
-  const [breed, setInfo] = useState<BreedType[]>([]);
+  const [breed, setBreed] = useState<BreedType[]>([]);
   const [errorMessage, setError] = useState<string>("");
```

Comment:
should write `setErrorMessage`
and you can not type this state, because it is primitive, just write `useState("");`